sklearn/utils/tests/test_utils.py

```diff
@@ -25,6 +25,9 @@
 from sklearn.utils.mocking import MockDataFrame
 from sklearn import config_context
 
+# toy array
+X = np.arange(9).reshape((3, 3))
```

Comment:
maybe name it something else (or even remove).

Since `X` is such a common name, it's hard from the tests to realize this is a global variable. And when we do, we need to check the top of the file to konw what it is